features/specs/gezag-meerderjarige/gezag-gedeeltelijke-bepaling.feature

```diff
@@ -0,0 +1,325 @@
+        * is ingeschreven in de BRP
+        * is meerderjarig
+        En de persoon 'Marcella' met burgerservicenummer '000000036'
+        * is in Nederland geboren
+        * is ingeschreven in de BRP
+        * is pas geboren
+
+  Regel: een meerderjarige die gezag heeft over een minderjarig samen met een ouder waarvan niet alle benodigde gegevens bekend zijn wordt gezag geleverd
+
+    # huidige gezag niet te bepaling melding: gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: ouder2 van bevraagde persoon is niet in BRP geregistreerd
+    Abstract Scenario: wanneer een ouder niet is ingeschreven in de BRP kan voor de andere ouder (ouder 1) alsnog gezag worden bepaald er is sprake van EenhoofdigOuderlijkGezag
+      Gegeven de persoon 'Joris' met burgerservicenummer '000000048'
+        * is meerderjarig
+        * is niet ingeschreven in de BRP
+      En persoon 'Marcella'
+        * heeft 'Pieternella' als ouder 1
+        * is erkend door 'Joris' als ouder 2 met erkenning bij geboorteaangifte
+      Als gezag wordt gezocht met de volgende parameters
+        | naam                | waarde                |
+        | burgerservicenummer | <burgerservicenummer> |
+      Dan heeft de response een persoon met de volgende gegevens
+        | naam                | waarde                |
+        | burgerservicenummer | <burgerservicenummer> |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+        | naam                             | waarde                                                                                                                           |
+        | type                             | EenhoofdigOuderlijkGezag                                                                                                         |
+        | minderjarige.burgerservicenummer | 000000036                                                                                                                        |      
+        | ouder.burgerservicenummer        | 000000012                                                                                                                        |
+        | toelichting                      | Gezag is niet volledig te bepalen omdat de gegevens van een ouder niet worden bijgehouden. Mogelijk heeft deze ouder ook gezag.  |
+
+      Voorbeelden:
+      | burgerservicenummer  |
+      | 000000012            |
+      | 000000036            |
+
+    # huidige gezag niet te bepaling melding:  gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: ouder1 van bevraagde persoon is niet in BRP geregistreerd
+    Abstract Scenario: wanneer een ouder niet is ingeschreven in de BRP kan voor de andere ouder (ouder 2) alsnog gezag worden bepaald er is sprake van EenhoofdigOuderlijkGezag
+      Gegeven de persoon 'Jorike' met burgerservicenummer '000000064'
+        * is meerderjarig
+        * is niet ingeschreven in de BRP
```

Comment:
wat betekent "is niet ingeschreven in de BRP" hier? De persoon heeft een bsn, dus is -als die niet in de BRP staat- ingeschreven in RNI.

Persoon is "pas geboren". Dat lees ik als na 1-1-2023. Als er van beide ouders een burgerservicenummer is, dus beide ouders staan in BRP of RNI, dan kan je gewoon tweehoofdig ouderlijk gezag leveren, toch?